src/rulesets/Base/array.jl

```diff
@@ -278,3 +278,129 @@ function rrule(::typeof(fill), value::Any, dims::Int...)
     end
     return fill(value, dims), fill_pullback
 end
+
+#####
+##### `findmax`, `maximum`, etc.
+#####
+
+for findm in (:findmin, :findmax)
+    findm_pullback = Symbol(findm, :_pullback)
+
+    @eval function frule((_, xdot), ::typeof($findm), x; dims=:)
+        y, ind = $findm(x; dims=dims)
+        ydot = (xdot[ind], NoTangent())
+        return (y, ind), Tangent{typeof((y, ind)),typeof(ydot)}(ydot)
```

Comment:
This is the same thing, but cleaner:
```suggestion
        return (y, ind), Tangent{typeof((y, ind))(xdot[ind], NoTangent())
```

This looks right to me.
we do not have as extensive practical tests of our forwards-mode as reverse mode.
So it is very possible there are problems with ChainRulesTestUtils for them.

Reply:
Ok, I tried leaving off the 2nd type parameter, but didn't try with a splat like this. But it does indeed work, done.